python/tidy/servo_tidy_tests/test_tidy.py

```diff
@@ -30,6 +32,18 @@ def test_tidy_config(self):
         self.assertEqual('invalid config table [wrong]', errors.next()[2])
         self.assertNoMoreErrors(errors)
 
+    def test_directory_checks(self):
+        dirs = {
+            "./python/tidy/servo_tidy_tests/dir_check/webidl_plus": ['webidl', 'test'],
+            "./python/tidy/servo_tidy_tests/dir_check/only_webidl": ['webidl'],
+            "./python/tidy/servo_tidy_tests/dir_check/empty": ['webidl']
```

Comment:
There's a `base_path` on the top of the file. We should make use of it and `os.path.join` with the relative paths where your test files reside.